artemis-server/src/main/java/org/apache/activemq/artemis/core/server/cluster/impl/BridgeImpl.java

```diff
@@ -251,22 +243,22 @@ public long sequentialID() {
 
    @Override
    public synchronized void start() throws Exception {
-      if (started) {
-         return;
+      if (state == State.STARTING || state == State.STARTED || state == State.STOPPING || state == State.PAUSING) {
+         logger.debug("Bridge {} state is {}. Ignoring call to start.", configuration.getName(), state);
+         if (state == State.STOPPING || state == State.PAUSING) {
+            throw ActiveMQMessageBundle.BUNDLE.bridgeOperationCannotBeExecuted(configuration.getName(), "started", state);
+         } else {
+            return;
+         }
```

Comment:
Probably better to read the state once and check, otherwise it could change between each of the checks and being read for the exception creation method reporting the state that caused it, e.g if it was pausing it might reach paused during the check+throw process and throw 'cant be started because paused'.

Reply:
Done.